### PR TITLE
DROOLS-4772: [DMN Designer] Open Properties panel by default

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-4772

### CODE REVIEW

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by manstis** [comment]
@jozef This PR is OK to review now. Thank-you.

**Comment by manstis** [comment]
Jenkins please execute full downstream build.

File: kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java [hunk 1]
```diff
@@ -247,6 +257,16 @@ public void onStartup(final ObservablePath path,
         decisionNavigatorDock.init(PerspectiveIds.LIBRARY);
     }
 
+    @OnOpen
+    public void onOpen() {
+        final String currentPerspectiveIdentifier = perspectiveManager.getCurrentPerspective().getIdentifier();
+        final Collection<UberfireDock> stunnerDocks = stunnerDocksHandler.provideDocks(currentPerspectiveIdentifier);
+        stunnerDocks.stream()
+                .filter(dock -> dock.getPlaceRequest().getIdentifier().compareTo(DiagramEditorPropertiesScreen.SCREEN_ID) == 0)
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java]: As we do not sort, I think we should use 
```java
Objects.equals(dock.getPlaceRequest().getIdentifier(), DiagramEditorPropertiesScreen.SCREEN_ID)
```

Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java]: Updated in new commit.. that'll teach me to copy and paste from BPMN!

**Comment by manstis** [comment]
@jomarko Hi, regarding your manual check and question as to whether the Properties dock should appear when switching between editors. This PR for DMN just replicates a requirement for BPMN/CM for consistency. Your question is quite valid however since DMN is just following the principle change for BPMN/CM I guess the answer needs to come from @romartin or @Josephblt who were involved with defining the original requirement.

**Comment by manstis** [comment]
@jomarko This PR can be re-reviewed following my last commit to make your [requested](https://github.com/kiegroup/kie-wb-common/pull/3021#discussion_r348308419) change.

**Comment by jomarko** [comment]
@manstis I required quite simple change, I am just waiting for @romartin and @Josephblt answer about switching between files.

**Comment by Josephblt** [comment]
Hello @jomarko. As I said in the [PR for BPMN](https://github.com/kiegroup/kie-wb-common/pull/3024#issuecomment-555788346) to keep the same functionality across all editor this should be included. So I vote yes for cherry picking this for 7.6 release. @romartin what do you think?

**Comment by manstis** [comment]
@Josephblt YOu're missing @jomarko point.

We know _this_ PR _should_ be merged to make DMN consistent with BPMN and CM.

However @jomarko comment was that should the Properties dock stay open when switching between different files? Please see his [video](https://issues.jboss.org/secure/attachment/12457669/switch-dmns.webm) that affects BPMN, CM and DMN.

**Comment by Josephblt** [comment]
Hello @jomarko, manstis. Ow, I see, sorry. I don't think so. Just the when you open it. If you just alternate they should stay in the same way they were.

**Comment by jomarko** [comment]
@manstis if I understand correctly @Josephblt 's answer, the behavior in video is an issue. So will we work on it as part of this PR or separately?

**Comment by manstis** [comment]
@jomarko This PR just applies the change already made to BPMN and CM to show the Properties Panel when a file is first opened to DMN. IMO the issue you have identified and confirmed by @Josephblt should be addressed by a new Stunner JIRA.

**Comment by Josephblt** [comment]
Hello @jomarko, @manstis. Yes I agree, this looks like an issue for another JIRA.

**Comment by jomarko** [comment]
@manstis do not work during PTO!

**Pull request merged**